src/pages/RFOX/components/Unstake/UnstakeStatus.tsx

```diff
@@ -82,7 +125,7 @@ export const UnstakeStatus: React.FC<UnstakeRouteProps> = () => {
         </AnimatePresence>
       )}
       <CardFooter flexDir='column' gap={2}>
-        <Button size='lg' variant='ghost'>
+        <Button as={Link} href={txLink} size='lg' variant='ghost'>
```

Comment:
We'll probably want to make this open a new window/tab.

Reply:
Added in https://github.com/shapeshift/web/pull/6957/commits/bccf2ea6f2f4e6511c7835c919cf6eec08efafcf.